internal/lib/sandbox/sandbox.go

```diff
@@ -348,6 +348,14 @@ func (s *Sandbox) SetNetworkStopped(createFile bool) error {
 
 func (s *Sandbox) createFileInInfraDir(filename string) error {
 	infra := s.InfraContainer()
+	// If the sandbox is not yet created,
+	// this function is being called when
+	// cleaning up a failed sandbox creation.
+	// We don't need to create the file, as there will be no
+	// sandbox to restore
+	if !s.created {
```

Comment:
Isn't this the same case as pointed out by @kolyshkin here: https://github.com/cri-o/cri-o/pull/4258#pullrequestreview-506918116 ?

Reply:
I'll rebase on https://github.com/cri-o/cri-o/pull/4241 once it merges :)